### PR TITLE
upgrade grafana to 4.1.1 and prometheus to v1.5.2

### LINKED ISSUE #120: Update prometheus/grafana to latest stable versions
So we can use the latest improvements.

### PR DESCRIPTION
**Pull request by tzach**
fix #120

### CODE REVIEW

**Comment by tzach** [comment]
@avikivity please review

File: start-all.sh [hunk 1]
```diff
@@ -50,10 +53,10 @@ fi
 
 if [ -z $DATA_DIR ]
 then
-    sudo docker run -d -v $PWD/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml:Z -p $PROMETHEUS_PORT:9090 --name $PROMETHEUS_NAME prom/prometheus:v1.0.0
+    sudo docker run -d -v $PWD/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml:Z -p $PROMETHEUS_PORT:9090 --name $PROMETHEUS_NAME prom/prometheus:$PROMETHEUS_VERSION
 else
     echo "Loading prometheus data from $DATA_DIR"
-    sudo docker run -d -v $DATA_DIR:/prometheus:Z -v $PWD/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml:Z -p $PROMETHEUS_PORT:9090 --name $PROMETHEUS_NAME prom/prometheus:v1.0.0
+    sudo docker run -d -v $DATA_DIR:/prometheus:Z -v $PWD/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml:Z -p $PROMETHEUS_PORT:9090 --name $PROMETHEUS_NAME prom/prometheus:PROMETHEUS_VERSION
```
Comment (amnonh) [inline start-all.sh]: you forgot the $ before the PROMETHEUS parameter

Comment (tzach) [reply start-all.sh]: Good catch! Surprisingly it did work (with the wrong version)

Comment (amnonh) [reply start-all.sh]: because it only takes effect when you use a local directory

**Comment by tzach** [comment]
@amnonh please try

**Pull request merged**